### PR TITLE
small footer updates

### PR DESCRIPTION
**Pull request by yonglid**

### CODE REVIEW

**Comment by yonglid** [comment]
added 1-3 sentence mission statement, removed documentation button, and changed join us to volunteer for us in footer

File: package.json [hunk 1]
```diff
@@ -22,7 +22,8 @@
     "build": "node build.js",
     "build:dev": "RUNTIME_API_HOST='https://zerobase-api-stage.herokuapp.com' RUNTIME_ENV=dev node build.js",
     "start:dev": "RUNTIME_ENV=dev node devServer.js",
-    "serve": "http-server ./public"
+    "serve": "http-server ./public",
+    "start": "node devServer.js"
```
Comment (JohnCLo) [inline package.json]: Hello friend, why did you add an npm start command here?

Comment (yonglid) [reply package.json]: Hi John, I'm on windows so the start:dev wasn't working for me. I created a regular start so I could run the site and make stylistic changes...could use some tips here.

**Comment by JohnCLo** [comment]
PR merges this changes and closes this out: https://github.com/zerobase-io/smart-tracing/pull/157
